page-tracking/src/page_info.rs

```diff
@@ -134,29 +142,68 @@ impl PageInfo {
         self.mem_type
     }
 
+    /// Returns if the page is locked.
+    pub fn is_locked(&self) -> bool {
+        self.locked
+    }
+
+    /// Obtains an exclusive reference to the page.
+    fn lock_page(&mut self) -> PageTrackingResult<()> {
```

Comment:
See my previous comment about the naming of these helpers: it's not clear what the difference is between `lock_page` vs `lock` based on the name. I'm not even sure you really need them anymore (i.e. you could just use `self.locked` directly), but if you do want to keep them I'd rename them to something like `do_lock` / `do_unlock`.

Reply:
Removed.

Reply:
Apologies...the gibhub interface isn't easy to navigate when there are several comments. Should be deleted for sure now.